### PR TITLE
EXP-614 Modificar textos MDR's para suas respectivas taxas na tela Detalhe de Transação

### PR DESCRIPTION
**Pull request by diegodsgarcia**
## Contexto
Atualmente as taxas estão sendo mostrado pelo texto "MDR", com isso realizando uma falha de comunicação com os clientes sobre as taxas de boleto e as taxas de pix. Esse PR tem o objetivo de corrigir esses textos com suas respectivas taxas.

## Issues linkadas
- [x] [EXP-614 Modificar textos MDR's para suas respectivas taxas na tela Detalhe de Transação](https://mundipagg.atlassian.net/secure/RapidBoard.jspa?rapidView=185&projectKey=EXP&modal=detail&selectedIssue=EXP-614)

## Como era antes
image
image

### Como é agora
image
image

## Como testar?
- Crie uma transação com Cartão, Boleto, Pix e verifique se os textos de taxa estão de acordo com seu respectivo tipo.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1c6d82c8fd0b2e9d384fe27dfeb820426304c789
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1c6d82c8fd0b2e9d384fe27dfeb820426304c789
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1c6d82c8fd0b2e9d384fe27dfeb820426304c789
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 1c6d82c8fd0b2e9d384fe27dfeb820426304c789
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/RecipientList/index.js [hunk 1]
```diff
@@ -86,8 +85,8 @@ class RecipientList extends Component {
           columns={installmentsTableColumns}
           installments={installments}
           key={key}
-          liabilities={liabilities}
-          liabilitiesLabel={liabilitiesLabel}
+          liabilities={liabilitiesLabel.liabilities}
```
Comment (vagnervst) [inline packages/pilot/src/containers/RecipientList/index.js:88]: da forma que está, estamos setando para os recebedores os mesmos liabilities indepentemente da configuração nas regras de split, aqui deveríamos continuar mostrando os valores de cada `recipient.liabilities` individualmente

talvez passar para cá a função `getLiabilitiesLabels` seja uma boa, nem precisaria se preocupar com a unicidade dos valores

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e7ec4304310b4aafbb308a9a99abd6fb0e48d010
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e7ec4304310b4aafbb308a9a99abd6fb0e48d010
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e7ec4304310b4aafbb308a9a99abd6fb0e48d010
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e7ec4304310b4aafbb308a9a99abd6fb0e48d010
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/RecipientList/index.js [hunk 2]
```diff
@@ -76,6 +77,10 @@ class RecipientList extends Component {
       const collapsed = notContains(index, expandedItems)
         && !expandAllRecipients
 
+      const getLiabilities = () => (
+        paymentMethod === 'credit_card' ? liabilities : [paymentMethod]
+      )
```
Comment (vagnervst) [inline packages/pilot/src/containers/RecipientList/index.js:82]: Repensando esse caso aqui, o mais correto seria adicionarmos o paymentMethod para quem está pagando o fee (split_rule.charge_transaction_fee: true), o que pensei então:
- Cartão de crédito - retorna as liabilities originais
- Cartão de débito - retorna as liabilities sem chargeback
- Pix/Boleto - retorna as liabilities sem chargeback, e com `mdr` renomeado para `pix` ou `boleto`

Fiz aqui uma implementação se quiser dar uma olhada:

```suggestion
      const replaceMdrWith = uncurryN(2, toReplace => map(
        when(
          equals('mdr'),
          always(toReplace)
        )
      ))

      const getLiabilities = () => {
        const isDebitTransaction = equals('debit_card')

        const isCardTransaction = either(
          equals('credit_card'),
          isDebitTransaction
        )

        const withoutChargeback = liabilities.filter(complement(
          equals('chargeback')
        ))

        if (!isCardTransaction(paymentMethod)) {
          return replaceMdrWith(paymentMethod, withoutChargeback)
        }

        if (isDebitTransaction) {
          return withoutChargeback
        }

        return liabilities
      }
```

Da forma como está agora, mostramos o pix ou boleto mesmo para recebedores com `charge_transaction_fee: false`

**Review by vagnervst — APPROVED** [review]

**Review by marisamodolo — APPROVED** [review]

**Pull request merged**